src/main/java/com/gooddata/GoodDataSettings.java

```diff
@@ -22,6 +24,7 @@
     private int connectionTimeout = secondsToMillis(10);
     private int connectionRequestTimeout = secondsToMillis(10);
     private int socketTimeout = secondsToMillis(60);
+    private String userAgent = StringUtils.EMPTY;
```

Comment:
I don't like to use null and try to avoid it where it's possible. And it's always better to use a constant...